pkg/cmd/workspace/code.go

```diff
@@ -107,11 +108,43 @@ var CodeCmd = &cobra.Command{
 		}
 
 		providerMetadata := ""
+
+		var yesFlag bool
```

Comment:
Please add a flag to the command that will allow the user to skip this prompt (e.g. `--auto-start`)

It's important that users have a way to run commands without opening the TUI

Reply:
done